tests/scripts/generate_pytest_test.py

```
@@ -0,0 +1,677 @@
+                        # Check if it's raising MissingRequiredArgumentError
+                        if isinstance(stmt.exc.func, ast.Name) and stmt.exc.func.id == "MissingRequiredArgumentError":
+                            # Extract the argument parameter
+                            for keyword in stmt.exc.keywords:
+                                if keyword.arg == "argument":
+                                    if isinstance(keyword.value, ast.Constant):
+                                        # Extract parameter name from "self.param_name"
+                                        param_str = keyword.value.value
+                                        if isinstance(param_str, str) and param_str.startswith("self."):
+                                            param_name = param_str[5:]  # Remove "self."
+                                            required_params.append(param_name)
+                break
```

Pylint (3.3.7)

[refactor] Too many nested blocks (9/5)

(R1702)